docs/excel/custom-functions-web-reqs.md

```diff
@@ -134,11 +134,9 @@ ws.onerror(error){
 
 ## Stream and cancel functions
 
-Streaming custom functions enable you to output data to cells that updates repeatedly, without requiring a user to explicitly refresh anything.
+Streaming custom functions enable you to output data to cells that updates repeatedly, without requiring a user to explicitly refresh anything. Streaming might be online or offline, such as a function which counts a number up by one each second (offline) or a function which requests data from the web at set intervals (online). To declare a streaming function, use the JSDoc comment tag `@stream`.
```

Comment:
Custom functions will not work when Excel is offline. I think this is not really necessary to specify. Any custom function can call the web, and they can be streaming, or not streaming. For example you could write all custom functions to call Azure functions that have the real calculation code. But they might all return a single value (not streaming). Also the header is odd. There is no such thing as a "cancel function" but the header sort of sounds like these are two different things.

Maybe for the H1 use "Make your streaming function cancelable" or similar.